docs/blog/2019-03-01-localization-with-gatsby-and-sanity/index.md

```diff
@@ -0,0 +1,272 @@
+---
+title: Localization with Gatsby and Sanity.io
+date: 2019-02-08
+author: Travis Stanley
+tags: ["sanity", "localization", "l10n"]
+image: "./images/gatsby-sanity-l10n.png"
+showImageInArticle: true
+---
+
+Localization is a common problem and there are _many_ solutions, each with their own benifits and trade-offs. With Gatsby and Sanity.io its possible to achieve a solution that is easy to work with and extend. If you haven't already read about why Gatsby and Sanity.io pair so well check out the blog post by [Knut Melvær](/contributors/knut-melvaer/): [Blazing fast development with Gatsby and Sanity.io](/blog/2019-01-25-blazing-fast-development-with-gatsby-and-sanity-io).
+
+I have completed two sites with Gatsby and Sanity that required localization. The first site needed to maintain every link from the original site without redirects. Each language living in its own subdomain, `es.my-web-page.com` for example. The simplest solution at the time was to use multiple builds which always bothered me. The second site I vowed to do it the _right™_ way.
+
+The new setup would generate pages for every language in one build step giving urls like `my-home-page.com/es/...`. This makes it easier for both the content team and our developers to use.
+
+## Setting Up Sanity to Support Translations
+
+Sanity.io has some good documentation which has great examples to help you achieve [localized](https://www.sanity.io/docs/localization) text.
+
+Create a new contentType schema called `localeText` to use with the schemas in your project.
+
+```js:title=/contentTypes/localeText.js
```

Comment:
Generally these are relative links, but I'm not really sure where this would live.

Is it co-located with the Gatsby app? If so:

```suggestion
```js:title=contentTypes/localeText.js
```

Reply:
Its actually located with your sanity project which _could_ be in the same place as your gatsby app but could also be anywhere else.

Reply:
Ah! Either way though, I think a relative link is the way to go. Generally the convention!